### PR TITLE
 #6314: Add standard project scripts to @mapstore/project

### PR DESCRIPTION
**Pull request by MV88**
## Description

This PR completes the implementations needed to make the [@mapstore/project](https://github.com/geosolutions-it/mapstore-project) work with the current master

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

#6792

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [ ] No

### CODE REVIEW

**Comment by MV88** [comment]
@mbarto @offtherailz can you do a final check, i have tested this in a local container and it was working

**Review by allyoucanmap — APPROVED** [review]

File: web/client/test-resources/mergeConfigs/02_austro_stateVars.patch.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      "type": "csw",
+      "title": "AustroControl Inspire CSW Service",
+      "autoload": true
+    },
+    "AustroControl Free CSW Service": {
+      "url": "https://sdimd-free.austrocontrol.at/geonetwork/srv/eng/csw-free",
+      "type": "csw",
+      "title": "AustroControl Free CSW Service",
+      "autoload": true
+    },
```
Comment (offtherailz) [inline web/client/test-resources/mergeConfigs/02_austro_stateVars.patch.json:35]: Do not quote existing services in tests

File: web/client/test-resources/mergeConfigs/02_austro_stateVars.patch.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
+[
```
Comment (offtherailz) [inline web/client/test-resources/mergeConfigs/02_austro_stateVars.patch.json:1]: do not quote customer names

**Review by offtherailz — CHANGES_REQUESTED** [review]
Personally I didn't tested, but to parallelize the work, I leave my comments ( I leave my colleques the testing phase)

need to document
- [ ] new paths (directories changes, updating existing documentation)
- [ ] patch system 
- [ ] changes, if any, with back-end configs (I've seen adding ldap files, spring security, so update also this part if needed)
- [ ] build variables, possibly
- [ ] migration, if any

Remove from tests reference to austro or any existing service (also geo-solutions ones, if possible)

**Review by allyoucanmap — APPROVED** [review]

File: docs/developer-guide/mapstore-migration-guide.md [hunk 3]
```diff
@@ -20,6 +20,23 @@ This is a list of things to check if you want to update from a previous version
 - Optionally check also accessory files like `.eslinrc`, if you want to keep aligned with lint standards.
 - Follow the instructions below, in order, from your version to the one you want to update to.
 
+
+## Migration from 2021.01.01 to 2021.02.00
+
+See [this](https://github.com/geosolutions-it/MapStore2/pull/6738/files) pr for more details
```
Comment (offtherailz) [inline docs/developer-guide/mapstore-migration-guide.md:26]:
```suggestion

During this release MapStore changed a lot the project system. The first phase of this migration has been identified by [this](https://github.com/geosolutions-it/MapStore2/pull/6738/files) pull request. In this PR we are supporting the backward compatibility as much as possible. Anyway this migration guidelines will change accordingly to the new system soon. 
```

File: docs/developer-guide/mapstore-migration-guide.md [hunk 4]
```diff
@@ -20,6 +20,23 @@ This is a list of things to check if you want to update from a previous version
...
+**Some key files have been moved to `configs` folder like:**
+- localConfig.json
+- new.json
+- pluginsConfig.json
+- config.json 
+- simple.json 
+
+checkout changes to various pom.xml 
+- **pom.xml**
+- **web/pom.xml**
```
Comment (offtherailz) [inline docs/developer-guide/mapstore-migration-guide.md:37]:
```suggestion
Back-end has been reorganized 
In particular: 
- all the java code has been moved from `web/src/` to the `java/` and `product/` directories (and `release`, already existing).
- `mapstore-backend` has been renamed into `mapstore-services`.

Check the changes in pom.xml to update. (future evolution of the project will avoid you to keep your own copies of the pom files as much as possible, for this reasons these migration guidelines will change soon.)

- **pom.xml**
- **java/web/pom.xml**
```

**Review by offtherailz — CHANGES_REQUESTED** [review]
I reviewed only the documentation. Please apply my suggestions where possible.

File: docs/developer-guide/mapstore-migration-guide.md [hunk 5]
```diff
@@ -20,6 +20,23 @@ This is a list of things to check if you want to update from a previous version
 - Optionally check also accessory files like `.eslinrc`, if you want to keep aligned with lint standards.
 - Follow the instructions below, in order, from your version to the one you want to update to.
 
+
+## Migration from 2021.01.01 to 2021.02.00
+
+See [this](https://github.com/geosolutions-it/MapStore2/pull/6738/files) pr for more details
+
+**Some key files have been moved to `configs` folder like:**
```
[the remark this answers is not in the document]
Comment (MV88) [reply docs/developer-guide/mapstore-migration-guide.md]: yes they needs to be moved or the app will not work and when writing them i am using the vs code preview

**Review by allyoucanmap — APPROVED** [review]

**Pull request merged**